### ISSUE TITLE
Plumb in the kitchen sinks

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#667]
We need to work out the system for hot water to the two sinks.

We might be able to get hot water from the toilets?  We've currently got one water heater, which feeds both sinks - do we use that to feed both sinks in the new building, although the workshop sink will be quite a way from the kitchen sink, so does it make more sense to get a second water heater for the workshop?

**Comment by johnmckerrell** [comment]
We need to get a plumber to plumb the kitchen sinks in. I've emailed Simon Derwent to see if he can do it, if we've not heard by Monday we will need to just find a plumber ourselves. At this point we should probably remove the electric water heater that's in Gostins and just manage with the kettle for now. There is a water heater in the toilets in the Tapestry but it's pretty small so it might not be good for running all four sinks.

Arrangement of the kitchen sinks can be found on #690 we will need a cold water feed for the dishwasher and waste provision, although I have a feeling the ikea sinks include somewhere that we can attach the dishwasher waste.

**Comment by amcewen** [comment]
Looks like the cold water feed comes up on the South side, so it might be easier to run it to the kitchen on that side, as the joists run the right way

**Comment by johnmckerrell** [comment]
@amcewen where did you get that information? Simon was just in and we thought the feed would need to go from inside the accessible toilet and through the wall.

Simon Derwent (whose username I can't find) has said he'd be able to plumb this all in. We can leave the water heater until we leave but we should bring the kitchen stuff over early really. If we leave the heater until we remove the kitchen we don't need to block off the pipes, just turn off the isolator.

**Comment by amcewen** [comment]
From @magman2112, when he was doing the networking on Saturday.  I couldn't add a photo from my phone, but it's here...

![](https://c1.staticflickr.com/1/872/40339933094_d6496a8603_b.jpg)

You might be able to make out the white plastic water pipe running horizontally across the lifted floorboard.  I've added a note to the [original photo on Flickr](https://www.flickr.com/photos/amcewen/40339933094/in/datetaken/) to mark it (and you can see higher-resolution versions there too)

**Comment by johnmckerrell** [comment]
And through the magic of *actually being in the space* I can wander over and lift that floorboard up :-) Oddly that pipe appears to come out of the floor about a foot to the left of that picture and then disappears under a joist to the right. I think the easiest option will likely be to come off the feed in the accessible toilet as there's an isolator there too but we can make Simon aware. (I then wandered off to do that but he'd left and I got distracted)

First photo below is looking left, second looking right, orientation left up to the reader to guess.

![img_0911](https://user-images.githubusercontent.com/27243/38017188-336ef4d4-3269-11e8-8b97-57a0dbe63983.JPG)
![img_0912](https://user-images.githubusercontent.com/27243/38017198-395daf7a-3269-11e8-9a43-3da0bb7a51f7.JPG)
